### ISSUE TITLE
Auto-fix for `unicorn/import-style`?

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#963]
```js
import {dirname} from 'path';

const foo = dirname(bar)
```

->

```js
import path from 'path';

const foo = path.dirname(bar)
```

I think it's doable.

**Comment by zanminkian** [comment]
If there is a variable name the same as module, auto-fix is dangerous.

```js
import {join} from 'path';

const path = join('/x', 'y');
```

**Comment by sindresorhus** [comment]
Auto-fix can easily detect conflicts and not auto-fix then or use a different name.

**Comment by sindresorhus** [comment]
Thinking more about this, I don’t think we should add an autofix. The rule supports arbitrary configured modules, and changing named imports into default or namespace-style property access is not guaranteed to preserve runtime behavior. It would also need a lot of special-case handling for aliases, name conflicts, comments, mixed imports, reexports, dynamic imports, and `require`.

Keeping this report-only is the simpler and safer
